### PR TITLE
Issue#2043 AddRule function

### PR DESCRIPTION
**Pull request by AntonyBang**
New function AddRule which add new rule to the db and save changes

### CODE REVIEW

File: RedirectApplication/RedirectApplication/Controllers/ValuesController.cs [hunk 1]
```diff
@@ -5,6 +5,8 @@
 using System.Web;
 using NGeoIP;
 using NGeoIP.Client;
+using RedirectApplication.RedirectDB;
```
Comment (jacob-l) [inline RedirectApplication/RedirectApplication/Controllers/ValuesController.cs]: Are you sure, that it is necessary there

Comment (AntonyBang) [reply RedirectApplication/RedirectApplication/Controllers/ValuesController.cs]: it allows to use AddRule

Comment (jacob-l) [reply RedirectApplication/RedirectApplication/Controllers/ValuesController.cs]: Usually, it is bad practice to keep unused usings. We add them, when it is used in current file

File: RedirectApplication/RedirectApplication/RedirectDB/RedirectRepository.cs [hunk 2]
```diff
@@ -16,5 +17,11 @@ public Boolean IsRuleExist(String Url)
         {
             return context.RedirectRules.Any(x => x.TargetUrl == Url);
         }
+
+        public void AddRule(RedirectRule NewRule)
```
Comment (maxmalov) [inline RedirectApplication/RedirectApplication/RedirectDB/RedirectRepository.cs:21]: Where this part of code is supposed to be used? Right now it looks like it is a dead code

**Pull request merged**